tests/test-parse-xml2.cpp

```diff
@@ -128,14 +128,8 @@ void assert_equal(uint64_t actual, uint64_t expected) {
 }
 
 int main(int argc, char *argv[]) {
-  char *srcdir = getenv("srcdir");
 
-  if (srcdir == nullptr) {
-    std::cerr << "$srcdir not set!\n";
-    return 1;
-  }
-
-  std::string inputfile = std::string(srcdir) + std::string("/tests/test_multipolygon.osm");
+  std::string inputfile = "tests/test_multipolygon.osm";
```

Comment:
It was in environment, cmakelists.txt in our case. All other tests did not use it.